jest.config.js

```diff
@@ -10,5 +10,6 @@ module.exports = {
     '^clappr$': '<rootDir>/node_modules/@clappr/core/dist/clappr-core.js',
     '^clappr-zepto$': 'clappr-zepto/zepto.js',
   },
+  testEnvironment: 'jsdom',
```

Comment:
Are there any tests needing the JSDOM Environment?

Reply:
Upon updating, I noticed that Jest was asking to add the jsdom environment due to some tests. Here's an example: 'should trigger a playback error if source load failed'.